docs/guide/user-guide.md

```diff
@@ -1 +1,109 @@
 # User guide
+
+In the user guide you can find a description of all components belonging to 
+DeepESDL.
+
+There are three components of DeepESDL: 
+1. [DeepESDL JupyterLab](#deepesdl-jupyterlab)
+2. [DeepESDL xcube viewer](#deepesdl-xcube-viewer)
+3. [DeepESDL datacube resources](#deepesdl-datacube-resources)
```

Comment:
4. [DeepESDL data cube generation] (# deepesdl-datacube-generation)

Here, we describe our general approach of data cube generation and refer to i) example notebooks for ad hoc generation by users and ii) recipes for published, reproducible cubes and the cube specification.